workflow-templates/git-slice.yml

```diff
@@ -78,7 +81,7 @@ jobs:
           git branch
           git checkout ${{ github.event.inputs.branch }}
           git pull origin ${{ github.event.inputs.branch }}
-          git merge origin/master
+          git merge origin/$GIT_SLICE_DEFAULT_BRANCH
           git push origin ${{ github.event.inputs.branch }}
```

Comment:
@rowlandekemezie Do we really need to do this here? Since push action already depends  on pull action

Reply:
I think the goal is to merge default branch to the branch inside the **CI**, then update the branch on Github before pushing to the client. That way we're pushing an updated branch to the client all the time.